[2024-05-15T12:26:33.036+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_ademe_data.drop_duplicates manual__2024-05-15T12:26:21.137656+00:00 [queued]>
[2024-05-15T12:26:33.051+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_ademe_data.drop_duplicates manual__2024-05-15T12:26:21.137656+00:00 [queued]>
[2024-05-15T12:26:33.052+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-05-15T12:26:33.084+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): drop_duplicates> on 2024-05-15 12:26:21.137656+00:00
[2024-05-15T12:26:33.096+0000] {standard_task_runner.py:60} INFO - Started process 3021 to run task
[2024-05-15T12:26:33.110+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'get_ademe_data', 'drop_duplicates', 'manual__2024-05-15T12:26:21.137656+00:00', '--job-id', '319', '--raw', '--subdir', 'DAGS_FOLDER/load.py', '--cfg-path', '/tmp/tmpryuuo6ua']
[2024-05-15T12:26:33.115+0000] {standard_task_runner.py:88} INFO - Job 319: Subtask drop_duplicates
[2024-05-15T12:26:33.241+0000] {task_command.py:423} INFO - Running <TaskInstance: get_ademe_data.drop_duplicates manual__2024-05-15T12:26:21.137656+00:00 [running]> on host 583e99962ff2
[2024-05-15T12:26:33.503+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='get_ademe_data' AIRFLOW_CTX_TASK_ID='drop_duplicates' AIRFLOW_CTX_EXECUTION_DATE='2024-05-15T12:26:21.137656+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-15T12:26:21.137656+00:00'
[2024-05-15T12:26:33.597+0000] {load.py:112} ERROR - An error occurred while dropping duplicates: name 'text' is not defined
[2024-05-15T12:26:33.599+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-05-15T12:26:33.614+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=get_ademe_data, task_id=drop_duplicates, execution_date=20240515T122621, start_date=20240515T122633, end_date=20240515T122633
[2024-05-15T12:26:33.646+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-05-15T12:26:33.669+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
